src/hooks/useSnippets.ts

```diff
@@ -14,8 +14,10 @@ export const useSnippets = () => {
     `/data/${slugify(language.lang)}.json`
   );
 
-  const fetchedSnippets = data
-    ? data.find((item) => item.categoryName === category)?.snippets
+  const fetchedSnippets: SnippetType[] = data
```

Comment:
instead of adding nested ternary operator, it would be better to populate it through a function.

Reply:
I did the thing 🫡